tests/test_app/components.py

```diff
@@ -44,6 +44,20 @@ def parameterized_component(x, y):
     )
 
 
+SimpleButtonModule = web.module_from_url(
```

Comment:
We want this to specifically be `module_from_file` rather than from URL because we want to exercise web modules loaded from [this view](https://github.com/idom-team/django-idom/blob/main/src/django_idom/http/views.py#L12).

Reply:
It has been switched.